apps/admin-gui/src/app/shared/components/dialogs/update-application-form-dialog/update-application-form-dialog.component.html

```diff
@@ -5,10 +5,30 @@
   <div *perunWebAppsLoader="loading; indicator: spinner">
     <h1 mat-dialog-title>{{'DIALOGS.UPDATE_APPLICATION_FORM.TITLE' | translate}}</h1>
     <div class="dialog-container" mat-dialog-content>
-      <mat-form-field>
-        <mat-label>{{'DIALOGS.UPDATE_APPLICATION_FORM.MODULE_NAME' | translate}}</mat-label>
-        <input matInput [(ngModel)]="moduleName" />
-      </mat-form-field>
+      <div
+        *ngFor="let moduleName of moduleNames; let i = index; trackBy: trackByIndex"
+        class="d-flex flex-row align-items-center">
+        <mat-form-field class="w-100 mb-n2">
+          <mat-label>{{'DIALOGS.UPDATE_APPLICATION_FORM.MODULE_NAME' | translate}}</mat-label>
+          <input [(ngModel)]="moduleNames[i]" matInput />
+        </mat-form-field>
+        <button
```

Comment:
I would prefer for the buttons to be green and red, to match how it is with policy collections 
![Screenshot from 2023-09-20 13-17-17](https://github.com/CESNET/perun-web-apps/assets/97669007/a5874ac3-35e6-41c9-922c-b6de6f58d7e1)